rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/azure/AzureBakeHandler.groovy

```diff
@@ -73,7 +76,29 @@ public class AzureBakeHandler extends CloudProviderBakeHandler{
 
   @Override
   Map buildParameterMap(String region, Object virtualizationSettings, String imageName, BakeRequest bakeRequest) {
-    return null
+
+    def selectedImage = azureBakeryDefaults?.baseImages?.find {it.baseImage.id == bakeRequest.base_os}
+
+    // TODO(larrygug): Presently rosco is only supporitng a single account. Need to update to support a named account
+    def selectedAccount = azureConfigurationProperties?.accounts?.get(0)
+
+    def parameterMap = [
+      azure_client_id: selectedAccount?.clientId,
+      azure_client_secret: selectedAccount?.appKey,
+      azure_resource_group: selectedAccount?.packerResourceGroup,
+      azure_storage_account: selectedAccount?.storageAccount,
+      azure_subscription_id: selectedAccount?.subscriptionId,
+      azure_tenant_id: selectedAccount?.tenantId,
+
```

Comment:
Since this is the only comment, I'll merge this as-is so I don't hold you up. Please fix the formatting in a follow-on PR though.